syntax = "proto3";
 
package transport_router_serialize;

message RoutingSettings {
  double bus_wait_time = 1;
  double bus_velocity = 2;
}